.github/workflows/merge_to_main_production.yml

```
@@ -49,6 +49,7 @@ env:
   TF_VAR_client_vpn_access_group_id: ${{ secrets.PRODUCTION_CLIENT_VPN_ACCESS_GROUP_ID }}
   TF_VAR_client_vpn_saml_metadata: ${{ secrets.PRODUCTION_CLIENT_VPN_SAML_METADATA }}
   TF_VAR_client_vpn_self_service_saml_metadata: ${{ secrets.PRODUCTION_CLIENT_VPN_SELF_SERVICE_SAML_METADATA }}
+  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ART_PAT }}
```

I think of Pat when I see this... but also there might be a typo
```suggestion
  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ARC_PAT }}
```